h/static/scripts/util/search-text-parser.js

```diff
@@ -106,39 +122,7 @@ function getLozengeValuesAndIncompleteSearchTerms(queryString) {
   };
 }
 
-/**
- * Function which returns individual lozenge from a string.
- *
- * @param {String} queryString A string of query terms.
```

Comment:
Nit: Primitive types are written with a lowercase name in JSDoc (eg. `string`, `number`, `boolean`, `null`, `undefined`).